src/status_im/common/signals/events.cljs

```diff
@@ -33,61 +38,75 @@
-      "message.delivered"          (let [{:keys [chatID messageID]} (js->clj event-js
-                                                                             :keywordize-keys
-                                                                             true)]
-                                     (models.message/update-message-status cofx
-                                                                           chatID
-                                                                           messageID
-                                                                           :delivered))
-      "mailserver.changed"         (mailserver/handle-mailserver-changed cofx (.-id event-js))
-      "mailserver.available"       (mailserver/handle-mailserver-available cofx (.-id event-js))
-      "mailserver.not.working"     (mailserver/handle-mailserver-not-working cofx)
-      "discovery.summary"          (summary cofx (js->clj event-js :keywordize-keys true))
-      "mediaserver.started"        {:db (assoc db :mediaserver/port (.-port event-js))}
-      "wakuv2.peerstats"           (wakuv2-peer-stats cofx (js->clj event-js :keywordize-keys true))
-      "messages.new"               (messages.transport/sanitize-messages-and-process-response cofx
-                                                                                              event-js
-                                                                                              true)
-      "wallet"                     (rf/dispatch [:wallet/signal-received event-js])
-      "local-notifications"        (local-notifications/process cofx
-                                                                (js->clj event-js :keywordize-keys true))
-      "community.found"            (link-preview/cache-community-preview-data (js->clj event-js
-                                                                                       :keywordize-keys
-                                                                                       true))
-      "status.updates.timedout"    (visibility-status-updates/handle-visibility-status-updates
-                                    cofx
-                                    (js->clj event-js :keywordize-keys true))
-      "localPairing"               (pairing/handle-local-pairing-signals
-                                    cofx
-                                    (js->clj event-js :keywordize-keys true))
-      "curated.communities.update" (rf/dispatch [:fetched-contract-communities
-                                                 (js->clj event-js :keywordize-keys true)])
-      "waku.backedup.profile"      (rf/dispatch [:profile/update-profile-from-backup
-                                                 (js->clj event-js :keywordize-keys true)])
-      "waku.backedup.settings"     (rf/dispatch [:profile/update-setting-from-backup
-                                                 (js->clj event-js :keywordize-keys true)])
+      "wallet"
```

Comment:
do we want to have some comment here about order being  of some importance?

Reply:
It won't hurt to add a comment 👍🏼 Let me add it